tests/locator_test.go

```diff
@@ -296,11 +296,31 @@ func TestLocatorInputValue(t *testing.T) {
 		require.Equal(t, "text area", p.Locator("textarea", nil).InputValue(nil))
 	})
 	t.Run("ok/select", func(t *testing.T) {
-		require.Equal(t, "option text", p.Locator("select", nil).InputValue(nil))
+		require.Equal(t, "option text", p.Locator("#selectElement", nil).InputValue(nil))
 	})
 	t.Run("strict", func(t *testing.T) {
 		require.Panics(t, func() {
 			p.Locator("input", nil).InputValue(nil)
 		}, "should not select multiple elements")
 	})
 }
+
+//nolint:tparallel
+func TestLocatorSelectOption(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withFileServer())
+	p := tb.NewPage(nil)
+	require.NotNil(t, p.Goto(tb.staticURL("/locators.html"), nil))
+
+	t.Run("ok", func(t *testing.T) {
+		rv := p.Locator("#selectElement", nil).SelectOption(tb.toGojaValue(`option text 2`), nil)
+		require.Len(t, rv, 1)
+		require.Equal(t, "option text 2", rv[0])
+	})
```

Comment:
I knew you would ask me that 😊 And, you're right, it's fair. As I said in the PR description, I believe it should be in the `ElementHandle` test, not here. Because we only check whether we route calls to the `Frame`. WDYT?

Reply:
I think we'll probably forget to add it to `ElementHandle`, but OK.

And I guess it's fine if we leave non-matching selector tests for later. It might be more work for you that way, but it's up to you.